src/server/backend.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, *args, memory_cache: MemoryCache, **kwargs):
         for name, buf in self.module.named_buffers():
             assert not buf.requires_grad, f"Bloom layer parameters must not accumulate gradients, but {name} does"
 
-        self.inference_pool = TaskPool(self.inference_step, max_batch_size=1, name=f"{self.name}_inference")
+        self.inference_pool = TaskPool(self.inference_step, max_batch_size=4096, name=f"{self.name}_inference")
```

Comment:
This can have an adverse effect of grouping together concurrent requests for inference into one pytorch call. The current inference code will break in that case.